src/object/conveyor_belt.hpp

```diff
@@ -0,0 +1,91 @@
+//  SuperTux
+//  Copyright (C) 2022 Raoul1808 <raoulthegeek@gmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef HEADER_SUPERTUX_OBJECT_CONVEYOR_BELT_HPP
+#define HEADER_SUPERTUX_OBJECT_CONVEYOR_BELT_HPP
+
+#include "scripting/conveyor_belt.hpp"
+#include "squirrel/exposed_object.hpp"
+#include "supertux/moving_object.hpp"
```

Comment:
Move these two includes to the top, separated by an empty line from the current ones, as they are base classes for `ConveyorBelt`.